extensions/wikia/SponsorshipDashboard/sources/SDSourceGapi.class.php

```diff
@@ -404,8 +404,7 @@ protected function getGAFilters() {
 	protected function getGAUrlFromCityId() {
 
 		$url = WikiFactory::getWikiByID( $this->getCityId() )->city_url;
-		$hostname = str_replace( 'http://', '', $url );
-		$hostname = str_replace( '/', '', $hostname );
+		$hostname = parse_url( $url )[ 'host' ];
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$hostname". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)

[project-id]: wikia:app